model/src/catalog_parser.cpp

```diff
@@ -118,7 +118,6 @@ bool ParsePlugin(const std::string& xml, PluginMetadata& plugin) {
 
 
 bool ParseCatalog(const std::string xml, CatalogCtx* ctx) {
-  bool ok = true;
```

Comment:
This variable was not used.

Reply:
So why is it there? Unused variables should be removed.

Reply:
I don't know why it was there in the first place, but I did remove it. This is just a fly-by cleanup. I was commenting on why I removed it.